workers/generic-worker/multiuser_darwin.go

```diff
@@ -24,6 +25,53 @@ func defaultTasksDir() string {
 }
 
 func PreRebootSetup(nextTaskUser *gwruntime.OSUser) {
+
+	homeDir := filepath.Join("/Users", nextTaskUser.Name)
+	logPath := filepath.Join(homeDir, "gw-launch-agent.log")
+	LibraryDir := filepath.Join(homeDir, "Library")
```

Comment:
Use lowercase just for clarity.

```suggestion
	libraryDir := filepath.Join(homeDir, "Library")
```